plugins/tracers/gstctf.c

```diff
@@ -470,11 +475,33 @@ file_parser_handler (gchar * line)
   strcpy (ctf_descriptor->env_dir_name, line);
 }
 
+static gboolean
+verify_numeric_string (const gchar * str_number)
```

Comment:
I think `g_ascii_strtoull` already handles conversion errors. According to the docs:

```
If the string conversion fails, zero is returned, and endptr returns nptr (if endptr is non-NULL).
```

I think we can leverage that instead of re-checking all the string again.